sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificateClientOptions.cs

```diff
@@ -8,5 +8,45 @@ namespace Azure.Security.KeyVault.Certificates
 {
     public class CertificateClientOptions : ClientOptions
     {
+        /// <summary>
+        /// The latest service version supported by this client library.
+        /// For more information, see 
+        /// <see href="https://docs.microsoft.com/en-us/rest/api/keyvault/key-vault-versions"/>
+        /// </summary>
+        internal const ServiceVersion LatestVersion = ServiceVersion.V0_7_0;
+
+        /// <summary>
+        /// The versions of Azure Key Vault supported by this client
+        /// library. 
+        /// </summary>
+        public enum ServiceVersion
+        {
+#pragma warning disable CA1707 // Identifiers should not contain underscores
+            /// <summary>
+            /// The Key Vault API version 7.0.
+            /// </summary>
+            V0_7_0 = 0
```

Comment:
```suggestion
            V7_0 = 0
```

Reply:
[This](https://www.powershellgallery.com/packages/Az.KeyVault/0.7.0) made me think that it might roll over to a 1.0.0 from 0.7.0, but I wasn't able to confirm this in Azure docs.

Go with 7_0 instead?

Reply:
But in this case service version is 7.0[.0], not 0.7.0